src/main/java/org/apache/sysml/parser/BuiltinFunctionExpression.java

```diff
@@ -1000,6 +1000,37 @@ else if(in.length == 3)
 			output.setBlockDimensions(id.getRowsInBlock(), id.getColumnsInBlock()); 
 			break;
 			
+		case CONV2D:
+		case CONV2D_BACKWARD_FILTER:
+		case CONV2D_BACKWARD_DATA:
+		case MAX_POOL:
+		case AVG_POOL:
+		case MAX_POOL_BACKWARD:
+		{
+			// At DML level:
+			// output = conv2d(input, filter, input_shape=[3, 2, 2], filter_shape=[3, 2, 2], 
+			// strides=[1, 1], border_mode="valid")
```

Comment:
Side note... It would be really nice to have an arbitrary list construct in DML to hold various items, such as multiple matrices to be passed around.